apollo-router/src/services/layers/apq.rs

```diff
@@ -79,6 +83,7 @@ async fn apq_request(
                 request.supergraph_request.body_mut().query = Some(cached_query);
                 Ok(request)
             } else {
+                let _ = request.context.insert("persisted_query_miss", true);
```

Comment:
Is it still used ? I can't find the corresponding get. But if it's still relevant then please use a constant with some namespacing like `apq::persisted_query_miss`